packages/core/src/browser/style/status-bar.css

```diff
@@ -93,6 +93,10 @@ body.theia-no-open-workspace #theia-statusBar {
     background-color: var(--theia-statusBarItem-offlineActiveBackground) !important;
 }
 
+.theia-mod-debugging #theia-statusBar .area .element {
+    color: var(--theia-statusBar-debuggingForeground);
+}
```

Comment:
@ChayaLau the idea is good, but the implementation should not be in `core` but rather `debug` (as it is debug specific):

https://github.com/eclipse-theia/theia/blob/07aacaa3fa12dc517550293796cc17aae33e90b8/packages/debug/src/browser/style/index.css#L421-L426